mkdocs.yml

```diff
@@ -17,6 +17,7 @@ nav:
     - Environment: development/environment.md
     - Guidelines: development/guidelines.md
```

Comment:
Should we call this `Project Guidelines` instead?

Reply:
I think it makes sense. Changed it.